jsk_robot_common/jsk_robot_startup/lifelog/common_logger.launch

```diff
@@ -277,6 +280,32 @@
       </rosparam>
     </node>
 
+    <!-- face save aws face data -->
+    <node if="$(arg save_faces)"
+          name="face_logger"
+          pkg="jsk_robot_startup" type="mongo_record.py"
+          machine="$(arg machine)"
+          respawn="$(arg respawn)">
+      <rosparam subst_value="true">
+        topics:
+        - /aws_auto_checkin_app/class
+        - /aws_detect_faces/attributes
+      </rosparam>
+    </node>
+
+    <!-- face dialogflow I/O -->
+    <node if="$(arg save_dialogflow)"
+          name="dialogflow_logger"
+          pkg="jsk_robot_startup" type="mongo_record.py"
+          machine="$(arg machine)"
+          respawn="$(arg respawn)">
+      <rosparam subst_value="true">
+        topics:
+        - /dialogflow_client/text_action/goal
+        - /dialogflow_client/text_action/result
+      </rosparam>
+    </node>
```

Comment:
add `collection` arg in these nodes, too.

Reply:
Thank you for reviewing. I added in [dd0061f](https://github.com/jsk-ros-pkg/jsk_robot/pull/1728/commits/dd0061f19c70db31a2d5eca6d55254a58ca8ef43).